src/libs/freetype/lv_freetype.c

```diff
@@ -313,4 +352,67 @@ static void lv_freetype_drop_face_id(lv_freetype_context_t * ctx, FTC_FaceID fac
     LV_ASSERT_MSG(false, "face_id not found");
 }
 
+/*-----------------
+ * Cache Node Cache Callbacks
```

Comment:
Could you comment on what's a cache node cache?

Reply:
@FASTSHIFT named this name before